### ISSUE TITLE
Unison couldn't access a git repository

### ISSUE THREAD

**Issue by Sheyne** [unisonweb/unison#1153]
Similar to #1152, I went to try playing with Unison and created the following devcontainer. https://github.com/Sheyne/unison These are very easy to use from VSCode [just open container in vscode with the containers plugin, and it'll config your dev env in the VM for consistent builds (more info: https://code.visualstudio.com/docs/remote/containers)].

I had trouble pulling the base both from the local FS and GitHub: 

```
.> pull https://github.com/unisonweb/base .base
  I couldn't access a git repository at https://github.com/unisonweb/base. Make sure the repo
  exists and that you have access to it.
```

```
# git clone https://github.com/unisonweb/base
Cloning into 'base'...
remote: Enumerating objects: 2664, done.
remote: Counting objects: 100% (2664/2664), done.
remote: Compressing objects: 100% (1822/1822), done.
remote: Total 2664 (delta 811), reused 2619 (delta 770), pack-reused 0
Receiving objects: 100% (2664/2664), 909.77 KiB | 1.20 MiB/s, done.
Resolving deltas: 100% (811/811), done.
Checking out files: 100% (4512/4512), done.
```

```
> pull file://workspaces/unison/base
  I couldn't access a git repository at file://workspaces/unison/base. Make sure the repo exists
  and that you have access to it.
.> 
```

```
# ls /workspaces/unison/base
README.md
```

**Comment by aryairani** [comment]
@Sheyne Could you tell us a bit about your environment?  (OS, etc; whatever you can think of.)

**Comment by Sheyne** [comment]
I'm running in a VM with my full configuration is described in https://github.com/Sheyne/unison/blob/master/.devcontainer/Dockerfile

**Comment by aryairani** [comment]
I'm not super familiar with Docker containers — what host OS and software do you use to run it?

**Comment by aryairani** [comment]
Which terminal emulator do you use to run Unison?

**Comment by aryairani** [comment]
Whoops, those last two messages were meant for your Unicode thread.

**Comment by aryairani** [comment]
> ```
> > pull file://workspaces/unison/base
>   I couldn't access a git repository at file://workspaces/unison/base. Make sure the repo exists
>   and that you have access to it.
> .> 
> ```
the `file://` uri scheme needs a third `/` to refer to an absolute path like `/workspaces/unison/base`, although there's clearly more going on here than that, as you should have been able to download it successfully with your initial Github uri.

Meanwhile, I was able to pull successfully from github using the Ubuntu Bionic livecd environment in mac virtualbox after `apt-get install git`
![image](https://user-images.githubusercontent.com/538571/72687241-f4e97700-3ac9-11ea-998c-51ad06df9e24.png)

Unfortunately I'm not sure yet what the relevant difference between your bionic docker vm vs the livecd environment might be to cause this issue, though I would love to know and get it resolved.

**Comment by aryairani** [comment]
(I do, however, see the same issue in my barebones docker vm as well.)
![image](https://user-images.githubusercontent.com/538571/72687343-e6e82600-3aca-11ea-971a-91ce32c5324b.png)

**Comment by aryairani** [comment]
Looks like it was something to do with no locale being set; setting a UTF-8 locale worked for me; see https://github.com/Sheyne/unison/pull/1.

I'm going to close this issue and #1152 for now, but feel free to reopen if you still have trouble.

**Comment by Sheyne** [comment]
Confirmed closed by Sheyne/unison#1.
